index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css">
+    <title>Yunus Ali</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <h4>Yunus Ali</h4>
+            <i class="bi bi-list"></i>
+        </nav>
+    </header>
+    <main>
+        <div>
+        <article>
+            <h2>I’m Yunus
+                Glad to see you!</h2>
+            I’m a software developer!
+            I can help you build a product ,
+            feature or website Look through
+            some of my work and experience!
+            If you like what you see and have
+            a project you need coded, don’t hestiate
+            to contact me.
+        </article>
+        <h4>Lets connect</h4>
+        <ul>
+            <li><i class="bi bi-twitter"></i></li>
+            <li><i class="bi bi-linkedin"></i></li>
+            <li><i class="bi bi-medium"></i></li>
+            <li><i class="bi bi-github"></i></li>
+            <li><i class="bi bi-slack"></i></li>
```

Comment:
I kindly suggest you to stick to Figma color, currently your design and Figma color are different:

Your design :
<img width="223" alt="image" src="https://user-images.githubusercontent.com/7243132/170326311-87987ffb-a462-4d9b-935c-8182e634d337.png">

Figma : 
<img width="223" alt="image" src="https://user-images.githubusercontent.com/7243132/170326254-a04812b6-ec29-41ef-b64a-e4cc4932fc26.png">